Scripts/script-ADIsUserMember.yml

```diff
@@ -3,14 +3,26 @@ commonfields:
   version: 1
 name: ADIsUserMember
 script: |-
+  res = []
+
   resp = demisto.executeCommand( 'ADGetUserGroups', demisto.args() )
```

Comment:
In the issue filed we also wanted the indirect membership support - this requires a change to the LDAP query being issued. Did you find out how to do this? I reckon it would require a change of approach to this entire script (otherwise we would have to enumerate recursively upwards which isn't very clean). If we can't do this using LDAP we will implement it by ourselves but let's find out first.